src/main/resources/default/templates/biz/scripting/mongo-scripts.html.pasta

```diff
@@ -30,6 +31,9 @@
                             <td>
                                 <a href="@apply('/scripting/scripts/%s', script.getIdAsString())">@script.getCode()</a>
                             </td>
+                            <td>
+                                @toUserString(script.isDisabled())
```

Comment:
**Optional:** Maybe add a `t:dot` so its easier to detect disabled scripts via the color of the dot?